### PR TITLE
convert website source to Markdown, print message from installer about Cygwin paths

### PR DESCRIPTION
**Pull request by johnyf**
These changes include:

- convert the TLAPS website sources from HTML to Markdown. The conversion from Markdown to HTML is now done using [`pandoc`](https://pandoc.org). A little HTML remains here and there inside the Markdown, to preserve some of the original formatting. The resulting website looks almost the same with before.

- move tutorial material from under the directory `doc/web/` to the directory `doc/doc/`.

- add a `Makefile` for the website, to:
  - copy specific files from under `doc/doc/` to under `doc/web/content/`
  - call a Python script that prepends HTML headers and appends HTML footers to Markdown sources, and then converts them to HTML files by calling `pandoc`

- add a message to the installer that informs about `PATH` updates that may be needed on Windows in order to call a `tlapm` that has been installed within Cygwin, from a Windows command-line environment

- use lowercase paths for website files, update the website links accordingly, and also the JavaScript script for generating HTML headers (specifically the webpage menus)

Please find more details about these changes in the commit messages.

### CODE REVIEW

**Review by damiendoligez — CHANGES_REQUESTED** [review]
I have built the new website on our Web server. I had to make two changes (see below) to get it to compile. The result is available here: http://tla.msr-inria.inria.fr/tlaps-new

If you have a look, you'll see that most links are broken because they are still in upper case, while you changed the file names to lower case.

**Comment by johnyf** [comment]
Thank you for reviewing the changes and building the website, I have updated the changes to correct the errors.

In the renaming of filepaths to lowercase, there were links in HTML that I had not renamed, and those caused the linking errors in the built website.

File: doc/web/assets/header.py [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+    key=len,
+    reverse=True)
+LOGO_IMG = (
+    'logo-MS-Research-Inria-Joint-Centre-Small.png')
+TEMPLATE_1 = f'''
+<div id="wrapper">
+  <div id="header">
+    <div id="title">
+      <a class="title"
+        href="{{base}}Home.html">
```
Comment (damiendoligez) [inline doc/web/assets/header.py:124]:
```suggestion
        href="{{base}}home.html">
```

File: doc/web/content/home.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+
+### Get it
+<div class="hr"></div>
+
+The current version of TLAPS is 1.4.5. It can (and should) be used from
+the [Toolbox IDE](
+    https://research.microsoft.com/en-us/um/people/lamport/tla/toolbox.html).
+TLAPS is free software, distributed under the BSD [license](
+    Download/License.html).
```
Comment (damiendoligez) [inline doc/web/content/home.md:33]:
```suggestion
    download/license.html).
```

**Review by damiendoligez — CHANGES_REQUESTED** [review]
There are still broken links, especially in the menus at the top and left of the page. Since they are generated from `MENUMAP` in `header.py`, I guess you'll have to add a function to lowercase the menu items.

**Comment by damiendoligez** [comment]
I think you'll need to run a link checker on this thing.

**Comment by damiendoligez** [comment]
I don't think we want to change our website technology at this point, and the installer program was removed some time ago.
Closing this PR without merging. Raise your hand if you disagree.

**Pull request closed without merging**